compiler-cli/src/new.rs

```diff
@@ -24,9 +24,14 @@ const ELIXIR_VERSION: &str = "1.15.4";
 #[derive(
     Debug, Serialize, Deserialize, Display, EnumString, VariantNames, ValueEnum, Clone, Copy,
 )]
-#[strum(serialize_all = "kebab_case")]
+#[strum(serialize_all = "lowercase")]
 pub enum Template {
-    Lib,
+    #[strum(serialize = "erlang", serialize = "erl")]
```

Comment:
Could you put back `Lib` please? To remove it would be a breaking change. Don't include it in the help text though

Reply:
Thank you. Remove the deprecation notice for this variant please.

The code for how these small string is super complicated now and very hard to understand. Can we simplify this please, perhaps just have a regular array/vector of values, or a string.

Reply:
Thanks for the feedback!
I've made the changes in the latest commit e8336a0c6382d2e87e0138d8a5d634a10603f6ca , only keeping the `clap` annotation for the help documentation.